Content.Server/Doors/Systems/DoorSystem.cs

```diff
@@ -174,6 +177,12 @@ private void OnDoorAltVerb(EntityUid uid, DoorComponent component, GetVerbsEvent
         });
     }
 
+    private void OnGetPryMod(EntityUid uid, DoorComponent component, DoorGetPryTimeModifierEvent args)
+    {
+        if (_power.IsPowered(uid))
+            args.PryTimeModifier *= component.PoweredPryModifier;
+    }
```

Comment:
you're subscribing to an event in the same system thats raising it. shouldn't this be in airlock?